### DISCUSSION TITLE
How to Show the Toggl Pane from the Demo GIF

### DISCUSSION THREAD

**Discussion by NeodymiumPhish** [mcndt/obsidian-toggl-integration#76] [Feature Requests]
Hey all! Loving this project. I used to use Toggl quite often for an old job, then dropped it because I had no use for it for the past 2 years. Now I'm in a role where I would benefit greatly from this sort of time tracking, and already have an Obsidian Vault configured about exactly how I like it. 

However, I can't figure out how to show the Toggl pane that's visible in the demonstration gif in the readme. Is that something that's custom-made or coming soon?

**Comment by mcndt** [comment]
Hi NeodymiumPhish!

The way the plugin is set up, the pane should be added automatically to the top right slot in the right sidebar.

If it is your first time loading the plugin, it is possible that the pane is added silently to the sidebar as a new tab that you have to switch to. The tab should have a clock icon.

If it doesn't appear, you can try reinstalling the plugin and restarting Obsidian.

Let me know if you still have trouble with this, it is perfectly possible that you discovered a bug!

**Reply by NeodymiumPhish** [reply]
Awesome, thanks! I reinstalled and still couldn't see it, but I guess that's because it was tied with whatever pane the links and calendar pane displayed with. Once I used the command palette to display the links pane, it showed up as one of the 4 attached panes! 

This is such an awesome tool, by the way! Thanks for setting this up and sharing it with us!
